### ISSUE TITLE
Selective Call - only first phone_number

### ISSUE THREAD

**Issue by karol-skibinski** [nigelm/broadworks_ocip#1]
* Broadworks OCI-P Interface version: 16
* Python version: 3.8
* Operating System: Windows 10

### Description
I trying to read Selective Call Acceptance Criteria. 
Output has only first phone_numer.

### What I Did

When I wrote:
```
...
res_group = api.command("UserSelectiveCallAcceptanceGetCriteriaRequest16",user_id="22XXXXXX3@vpbx.plus.pl", criteria_name="test")
j_group = res_group.to_json(indent=2)
print(j_group)
...

```
I have output with only first phone_number:
```
{
  "time_schedule": null,
  "holiday_schedule": null,
  "blacklisted": false,
  "from_dn_criteria": {
    "from_dn_criteria_selection": "Specified Only",
    "include_anonymous_callers": false,
    "include_unavailable_callers": false,
    "phone_number": "123456789"
}
```
But I have 2 (or more) phone numbers in XML stream:
```
<?xml version="1.0" encoding="ISO-8859-2"?>
<BroadsoftDocument protocol="OCI" xmlns="C" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<sessionId xmlns="">xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
</sessionId>
<command echo="" xsi:type="UserSelectiveCallAcceptanceGetCriteriaResponse16" xmlns="">
    <blacklisted>false
    </blacklisted>
    <fromDnCriteria>
        <fromDnCriteriaSelection>Specified Only
        </fromDnCriteriaSelection>
        <includeAnonymousCallers>false
        </includeAnonymousCallers>
        <includeUnavailableCallers>false
        </includeUnavailableCallers>
        <phoneNumber>123456789
        </phoneNumber>
        <phoneNumber>987654321
        </phoneNumber>
    </fromDnCriteria>
</command>
</BroadsoftDocument>
```

**Comment by nigelm** [comment]
I think this was fixed by the 1.5.0 release

**Comment by karol-skibinski** [comment]
I confirm, now (1.5.0) it's ok!
Thanks
